src/django/api/tests.py

```diff
@@ -600,6 +608,68 @@ def test_incorrect_country_code_raises_error(self, mock_get):
         )
 
 
+class FacilityAndProcessingTypeTest(TestCase):
```

Comment:
I know the issue said to create tests using a loop, but I found it easier to incrementally add functionality to the look up function with individual tests. I can condense if need be.